plugins/optimization-detective/storage/rest-api.php

```diff
@@ -73,6 +75,7 @@ function od_register_endpoint() {
 				return od_handle_rest_request( $request );
 			},
 			'permission_callback' => static function () {
+				return true;
```

Comment:
Tip: What I do during development is I have a mini plugin with the following filters:

```php
add_filter( 'od_url_metric_storage_lock_ttl', '__return_zero' );
add_filter( 'od_url_metric_freshness_ttl', '__return_zero' );
add_filter( 'od_url_metrics_breakpoint_sample_size', function () { return 1; } );
add_filter( 'od_can_optimize_response', '__return_true' );
```

Then you won't need this.

```suggestion
```

Reply:
Good idea! 😄